packages/core/src/App/Components/Elements/LandscapeBlocker/landscape-blocker.tsx

```diff
@@ -0,0 +1,48 @@
+import React from 'react';
+import { useLocation } from 'react-router-dom';
+import { isDisabledLandscapeRoute, isTabletOs, routes } from '@deriv/shared';
+import './landscape-blocker.scss';
+
+const LandscapeBlocker = () => {
+    const location = useLocation();
+    const pathname = location?.pathname;
+    const is_hidden_landscape_blocker = isDisabledLandscapeRoute(pathname);
+    const shouldShowDtraderTabletView = pathname === routes.root && isTabletOs;
```

Comment:
@farhan-nurzi-deriv please, change `routes.root` to `routes.trade` (I have the same issue in my PR).